.github/workflows/create-deployment-spack.yml

```diff
@@ -67,4 +67,10 @@ jobs:
           git -C ${{ env.ROOT_VERSION_LOCATION }} clone https://github.com/ACCESS-NRI/spack-config.git --branch ${{ inputs.spack-config-version }}
           ln -s -r -v ${{ env.ROOT_VERSION_LOCATION }}/spack-config/v${{ steps.strip.outputs.version-dir }}/${{ vars.DEPLOYMENT_TARGET }}/* ${{ env.ROOT_VERSION_LOCATION }}/spack/etc/spack/
           mkdir ${{ env.ROOT_VERSION_LOCATION }}/release
+
+          # Create restricted folders
+          mkdir -p ${{ env.ROOT_VERSION_LOCATION }}/restricted/ukmo/release
+          mkdir -p ${{ env.ROOT_VERSION_LOCATION }}/restricted/ukmo/source_cache
+          # Only accessible to people in vk83_w, ki32_mosrs
```

Comment:
Do we want to specify the projects in an an environment variable and loop over them, e.g.
```
for project in $ENV{RESTRICTED_PROJECTS}
do
    setfacl -m "g:${project}:r-X,d:g:${project}:r-X" ${{ env.ROOT_VERSION_LOCATION }}/restricted
done
setfacl -m "g:vk83:---,d:g:vk83:---" ${{ env.ROOT_VERSION_LOCATION }}/restricted
setfacl -m "other::---,d:other::---" ${{ env.ROOT_VERSION_LOCATION }}/restricted
```
But this is hard-coding `vk83` group for no access ... probably not worth the loop until you want to chuck in the `/scratch` location too and it might make sense.

Reply:
I feel like there wouldn't be enough projects to add in future for it to be worth it...unless we make those projects secrets

Reply:
True dat. It was not a whole-hearted recommendation.